File: tests/fixtures/discrimination/go/out_of_corpus/temp_convert.go
package main

import "fmt"

func celsiusToFahrenheit(c float64) float64 {
	return c*9/5 + 32
}

func fahrenheitToCelsius(f float64) float64 {
	return (f - 32) * 5 / 9
}

func main() {
	temps := []float64{0, 100, 37, -40}
	for i := 0; i < len(temps); i++ {
		f := celsiusToFahrenheit(temps[i])
		fmt.Printf("%.1fC = %.1fF\n", temps[i], f)
	}
}

File: tests/fixtures/discrimination/rust/out_of_corpus/temp_convert.rs
fn celsius_to_fahrenheit(c: f64) -> f64 {
    c * 9.0 / 5.0 + 32.0
}

fn fahrenheit_to_celsius(f: f64) -> f64 {
    (f - 32.0) * 5.0 / 9.0
}

fn print_table() {
    let temps = [0.0, 20.0, 37.0, 100.0];
    for t in temps {
        let f = celsius_to_fahrenheit(t);
        println!("{:.1}C = {:.1}F", t, f);
    }
}

fn main() {
    print_table();
}
